dev/tasks/homebrew-formulae/apache-arrow.rb

```diff
@@ -34,13 +34,13 @@ class ApacheArrow < Formula
   license "Apache-2.0"
   head "https://github.com/apache/arrow.git"
 
-  depends_on "boost" => :build
-  depends_on "cmake" => :build
-  depends_on "llvm" => :build
   depends_on "aws-sdk-cpp"
+  depends_on "boost" => :build
   depends_on "brotli"
+  depends_on "cmake" => :build
   depends_on "glog"
   depends_on "grpc"
+  depends_on "llvm" => :build
```

Comment:
Did you sort these `depends_on` manually?
The original order is correct because Homebrew uses `depends_on` with `:build` -> `depends_on` without `:build` order.